cli/.js/src/main/scala/aqua/ArgOpts.scala

```diff
@@ -42,15 +45,20 @@ object ArgOpts {
           case Right(exprSpan) =>
             val expr = exprSpan.mapK(spanToId)
 
-            val args = expr.args.collect {
+            val argsV = expr.args.collect {
               case LiteralToken(value, ts) =>
-                LiteralRaw(value, ts)
+                validNel(LiteralRaw(value, ts))
               case VarToken(name, _) =>
-                // TODO why BottomType?
-                VarRaw(name.value, BottomType)
-            }
-
-            validNel(CliFunc(expr.funcName.value, args, expr.ability.map(_.name)))
+                // TODO it will be changed after 'jsTypeToAqua'
```

Comment:
I don't understand what this comment means. You need something to do later? What and why? Or `jsTypeToAqua` is a name of a function that's to be called?